SECURITY.md

```diff
@@ -0,0 +1,34 @@
+# Security policy
+
+Thanks for helping make `setup-beam` safer for everyone.
```

Comment:
These should be rephrased for otp_builds :)

Reply:
Yeah 👍

Reply:
Fixed, squashed and force-pushed.